bfdd/ptm_adapter.c

```diff
@@ -352,6 +352,32 @@ static int _ptm_msg_read(struct stream *msg, int command, vrf_id_t vrf_id,
 	 * - c: profile name length.
 	 * - X bytes: profile name.
 	 *
+	 * New format (with session id and status):
```

Comment:
No need to replicate the **New format**, just update the previous section with the new fields.